common.go

```diff
@@ -1,4 +1,61 @@
 package sdk
 
+import (
+	"bytes"
+	"encoding/json"
+	"strings"
+
+	"gopkg.in/bblfsh/sdk.v1/protocol"
+)
+
 const NativeBin = "/opt/driver/bin/native"
 const NativeBinTest = "/opt/driver/src/build/native"
+
+
+// NativeParseResponseToString is used for pretty print a native response. Used
+// for tests and fixtures regenetation.
+func NativeParseResponseToString(res *protocol.NativeParseResponse) (string, error) {
```

Comment:
It'd make more sense to implement the Stringer interface instead of creating an ad-hoc method name for that.

Reply:
[Native]ParseResponse objects already have a String() method (automatically provided from gRPC I think); this function already was in the integration tests and uses a "prettier" format keeping the fields sorted and removing the Elapsed field.